src/main/java/com/zebrunner/mcloud/grid/MobileRemoteProxy.java

```diff
@@ -264,6 +283,65 @@ private boolean isVideoEnabled(TestSession session) {
         return isEnabled;
     }
     
+    private boolean isLogEnabled(TestSession session) {
```

Comment:
how about merging both log/video to the single method
isCapabilityEnabled(TestSession session, String capability)